<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KImageUI</class>
 <widget class="QWidget" name="KImageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="m_open">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset theme="document-open">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="m_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset theme="document-save">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="m_rotateleft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Rotate left</string>
     </property>
     <property name="icon">
      <iconset theme="object-rotate-left">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>217</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="m_rotateright">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Rotate right</string>
     </property>
     <property name="icon">
      <iconset theme="object-rotate-right">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="m_quit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
     <property name="icon">
      <iconset theme="application-exit">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="10">
    <widget class="QLabel" name="m_image">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="m_view">
     <item>
      <property name="text">
       <string>Ignore</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Keep</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Expanding</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QListWidget" name="m_thumbnail">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
